src/components/SearchGridForm.vue

```diff
@@ -10,14 +10,13 @@
       v-if="!isFilterVisible"
       class="button filter-toggle"
       type="button"
-      @click.prevent="onToggleSearchGridFilter()"
-      @keyup.enter.prevent="onToggleSearchGridFilter()"
+      @click.prevent="onToggleSearchGridFilter"
```

Comment:
Unrelated to your changes but why `.prevent`? Since the button has `type="button"`, it has no default action associated with it.

Reply:
Good catch, @dhruvkb ! I found why it had `.prevent`: because this button used to be an `a` link: https://github.com/WordPress/openverse-frontend/commit/21407eb40153f2dfb697cb9be5541770d41acffb#diff-42fd9b361230feaa7e4766d70160004df481bc85f3519588c860d09ea0ab9573R40